js/client-topbar.js

```diff
@@ -91,7 +91,7 @@
 			case 'rooms':
 				return buf + ' aria-label="Join chatroom"><i class="fa fa-plus" style="margin:7px auto -6px auto"></i> <span>&nbsp;</span></a></li>';
 			case 'battle':
-				var name = BattleLog.escapeHTML(room.title);
+				var name = room.title;
```

Comment:
Specifically, this is bad and will need to be reverted.

Reply:
```suggestion
				var name = BattleLog.escapeHTML(room.title);
```

Reply:
Turns out it was also unnecessary! The other was all that was needed to get the correct results.